<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TetrixWindowClass</class>
 <widget class="QMainWindow" name="TetrixWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>1256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TetrixWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="boardFrame">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>900</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="otherFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Adobe 宋体 Std L</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NEXT PIECE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nextPieceLabel">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>280</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Adobe 宋体 Std L</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>LEVEL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="levelNumber"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Adobe 宋体 Std L</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>LINES REMOVED</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lineNumber"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Adobe 宋体 Std L</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SCORE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="scoreNumber"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pauseButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>Pause</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TetrixWindow.qrc"/>
 </resources>
 <connections/>
</ui>
